packages/@uppy/audio/src/index.js

```diff
@@ -0,0 +1,366 @@
+const { h } = require('preact')
+const { UIPlugin } = require('@uppy/core')
+const getFileTypeExtension = require('@uppy/utils/lib/getFileTypeExtension')
+const supportsMediaRecorder = require('./supportsMediaRecorder')
+const RecordingScreen = require('./RecordingScreen')
+const PermissionsScreen = require('./PermissionsScreen')
+const locale = require('./locale.js')
+
+/**
+ * Audio recording plugin
+ */
+module.exports = class Audio extends UIPlugin {
+  static VERSION = require('../package.json').version
+
+  #stream = null
+
+  #audioActive = false
+
+  #recordingChunks = null
+
+  #recorder = null
+
+  #capturedMediaFile = null
+
+  constructor (uppy, opts) {
+    super(uppy, opts)
+    this.mediaDevices = navigator.mediaDevices
+    this.supportsUserMedia = this.mediaDevices != null
+    this.protocol = globalThis.location.protocol.match(/https/i) ? 'https' : 'http'
```

Comment:
Shouldn't we make those private? That doesn't look like something we'd like our users to tweak.

Reply:
Not crazy about even more lines with empty line in between at the start of the class, but yeah, I suppose we should. Thanks :)